recipes/lpsolve/run_test.bat

```diff
@@ -0,0 +1,11 @@
+:: test the command line interface
+lp_solve -mps plan.mps
+
+:: FIXME: the following section doesn't work because cl.exe isn't available
```

Comment:
This isn't currently supported, see https://github.com/conda/conda-build/issues/1059

I don't think it does any harm leaving it here in case the feature is available in the future. The library is already tested using ctypes.